components/ReferenceManager/basic_page_layout/BasicTogglableNavbarLeft.tsx

```diff
@@ -60,6 +60,25 @@ export default function BasicTogglableNavbarLeft({
   const currentOrg = getCurrentUserCurrentOrg();
   const [currentOrgProjects, setCurrentOrgProjects] = useState<any[]>();
   const router = useRouter();
+  const [childrenOpenMap, setChildrenOpenMap] = useState({});
+
+  useEffect(() => {
+    const idsOpen = window.localStorage.getItem("projectIdsOpen");
```

Comment:
let's nullchain this just in case window gets invoked at server level and breaks the FE
```suggestion
    const idsOpen = window?.localStorage?.getItem("projectIdsOpen") ?? false
```